infrastructure/bootstrap/src/main/resources/application.yaml

```diff
@@ -0,0 +1,29 @@
+spring:
+  application:
+    name: loyalty-campaign-service
+  lifecycle:
+    timeout-per-shutdown-phase: 60s
+  cloud:
```

Comment:
This picture was an example
we not use openfeign http client

I will provide config later